src/pages/create/recruitCreatePage/RecruitCreatePage.tsx

```diff
@@ -1,32 +1,30 @@
-import React, { useEffect } from 'react';
+import React from 'react';
 import S from './RecruitCreatePage.styled';
-import { TitleAndIntro } from '../../../components/index';
-import RecruitInfoWrapper from './wrappers/RecruitInfoWrapper';
-import RecruitPostWrapper from './wrappers/RecruitPostWrapper';
+import {
+	BasicInformation,
+	ControlButtons,
+	Description,
+	DetailedInformation,
+	RecruitRoles,
+} from '../../../components/index';
 import { useMutation } from '@tanstack/react-query';
-import { useRecoilValue, useRecoilState } from 'recoil';
+import { useRecoilState, useSetRecoilState, useRecoilValue } from 'recoil';
 import { recruitInputState, validState } from '../../../atom';
-import { postingRecruit } from '../../../api';
+import { postingRecruit } from '../../../service/recruit/posting';
 import { useNavigate } from 'react-router-dom';
-
-const descriptions = [
-	'함께할 멤버들에게 알릴 기본 정보들을 기입해주세요!',
-	'기본 정보를 기반으로 구인글을 제공할 수 있습니다.',
-];
-
-const introductions = [
-	'게시될 구인글을 작성해주세요!',
-	'어떤 방식으로 진행하고 싶은지 구체적으로 작성해주세요.',
-];
+import RecruitTags from '../../../components/recruit/create/RecruitTags';
+import { InputState } from '../../../types';
 
 const RecruitCreatePage = () => {
 	const navigate = useNavigate();
-	const recruitFormData = useRecoilValue(recruitInputState);
-	const [isSubmit, setIsSubmit] = useRecoilState(validState);
-	const onClickCancel = () => {
-		// 모달창 띄워서 한 번 더 확인시키고 이동하기
-		// navigate('/');
-	};
+	const setIsSubmit = useSetRecoilState(validState);
+	const [formData, setFormData] = useRecoilState<InputState>(recruitInputState);
```

Comment:
```suggestion
	const formData = useRecoilValue<InputState>(recruitInputState);
```
setFormData 안사용하시길래 이렇게 변경하는게 좋아보입니다!

Reply:
사용하고 있어요!!

```
if (postAvailable) {
	uploadPost.mutate(formData);
	navigate(`/recruit/${uploadPost}`);
}
```